examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -43,6 +43,8 @@
 # First, we retrieve data from nilearn provided (general-purpose) datasets
 # ------------------------------------------------------------------------
 
+import matplotlib.pyplot as plt
```

Comment:
out of curiosity: why is this new import needed?

Reply:
it's used to change the figure size in the modified plots in L122 and L134. Maybe better to move it below to where it's used